src/components/controls/TextArea/__tests__/TextArea.test.tsx

```diff
@@ -84,6 +69,65 @@ describe('TextArea', () => {
         });
     });
 
+    describe('error', () => {
+        test('render error message with error prop', () => {
+            const {container} = render(<TextArea error="Some Error" />);
+
+            // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
+            expect(container.querySelector('.g-text-area__error')).toBeInTheDocument();
+            expect(screen.getByText('Some Error')).toBeVisible();
+        });
+
+        test('render error message with errorMessage prop', () => {
+            const {container} = render(
+                <TextArea errorMessage="Some Error with errorMessage prop" />,
+            );
+
+            // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
+            expect(container.querySelector('.g-text-area__error')).toBeInTheDocument();
+            expect(screen.getByText('Some Error with errorMessage prop')).toBeVisible();
+        });
+
+        test('do not show error without error/errorMessage prop', () => {
+            const {container} = render(<TextArea />);
+
+            // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
+            expect(container.querySelector('.g-text-area__error')).not.toBeInTheDocument();
```

Comment:
U can use `data-qa` attributes for such purposes.
[How to set](https://github.com/gravity-ui/uikit/blob/main/src/components/Select/components/SelectControl/SelectControl.tsx#L139)
[How to use](https://github.com/gravity-ui/uikit/blob/main/src/components/Select/__tests__/Select.base-actions.test.tsx#L35)